docs/synthetics/http-monitor.md

```diff
@@ -74,6 +74,63 @@ By default, the UI adds the below conditions while creating an HTTP monitor. You
 * **Response Code** *equals* `200`
 * **Response Time** metric *less than* **20000 ms**
 
+
+### Configuring Custom Alerts based on JSON response
+In addition to our preconfigured alert conditions, you can also create your own based on the JSON returned in the response body. As mentioned above, this can be done using the **Response Body JSON** type, where [JSONPath](https://github.com/json-path/JsonPath) can be used to extract a specific value and compare it against the expected value.
```

Comment:
When I was writing Lucene in Action the editors regularly had beef with "position references" like "As mentioned above".  OK, that was a printed book, so above/below didn't make a sense in a physical book.  Does "As mentioned above" add value here?

Reply:
Just stating that we already mentioned this a bit earlier in case some user is thinking "hey you already covered this", while also having this section contain all the information it needs on its own for people who just skip to a particular section.